services/dynamodb/src/main/java/software/amazon/awssdk/services/dynamodb/model/ItemBuilder.java

```diff
@@ -0,0 +1,351 @@
+/*
+ * Copyright 2010-2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package software.amazon.awssdk.services.dynamodb.model;
+
+import static java.util.stream.Collectors.toList;
+import static software.amazon.awssdk.utils.CollectionUtils.toMap;
+
+import java.nio.ByteBuffer;
+import java.util.AbstractMap.SimpleImmutableEntry;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import software.amazon.awssdk.annotations.ReviewBeforeRelease;
+import software.amazon.awssdk.utils.builder.SdkBuilder;
+
+/**
+ * Used to build a DynamoDB item map.
+ *
+ * For example to pass to a {@link PutItemRequest.Builder#item(Map)}.
+ */
+@ReviewBeforeRelease("May want to do this a different way")
+public final class ItemBuilder implements SdkBuilder<ItemBuilder, Map<String, AttributeValue>> {
+
+    private Map<String, AttributeValue> item = new HashMap<>();
+
+    private ItemBuilder() {
+    }
+
+    /**
+     * Create a new instance of the {@link ItemBuilder}.
+     *
+     * @return a new instance of the {@link ItemBuilder}
+     */
+    public static ItemBuilder builder() {
```

Comment:
Sorry to keep harping on this, but I really feel this should be `create()` because it's creating an `ItemBuilder`. 

`builder()` - a method on a `Foo` to get a `Builder<Foo>`
`create()` - a method on a `Foo` to get a `Foo`

Reply:
OR, crazy idea, we make an `Item` that extends `HashMap<String, AttributeValue>`, and add a `builder()` method on it.

`DynamoDbRequest.builder().item(Item.builder()....build()).build()`

Reply:
and yep - see my comment - great minds think alike.